format/postgres/pg_control.go

```diff
@@ -0,0 +1,167 @@
+package postgres
+
+import (
+	"embed"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/format/postgres/common"
+	"github.com/wader/fq/format/postgres/flavours/pgpro10"
+	"github.com/wader/fq/format/postgres/flavours/pgpro11"
+	"github.com/wader/fq/format/postgres/flavours/pgpro12"
+	"github.com/wader/fq/format/postgres/flavours/pgpro13"
+	"github.com/wader/fq/format/postgres/flavours/pgpro14"
+	"github.com/wader/fq/format/postgres/flavours/pgproee10"
+	"github.com/wader/fq/format/postgres/flavours/pgproee11"
+	"github.com/wader/fq/format/postgres/flavours/pgproee12"
+	"github.com/wader/fq/format/postgres/flavours/pgproee13"
+	"github.com/wader/fq/format/postgres/flavours/pgproee14"
+	"github.com/wader/fq/format/postgres/flavours/postgres10"
+	"github.com/wader/fq/format/postgres/flavours/postgres11"
+	"github.com/wader/fq/format/postgres/flavours/postgres12"
+	"github.com/wader/fq/format/postgres/flavours/postgres13"
+	"github.com/wader/fq/format/postgres/flavours/postgres14"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+)
+
+//go:embed pg_control.md
+var pgControlFS embed.FS
+
+func init() {
+	interp.RegisterFormat(decode.Format{
+		Name:        format.PG_CONTROL,
+		Description: "PostgreSQL control file",
+		DecodeFn:    decodePgControl,
+		DecodeInArg: format.PostgresIn{
+			Flavour: "",
+		},
+	})
```

Comment:
See comment about pg_btree.go. Also wonder if `PostgresIn` whould be `PgControlIn` here? the arg struct type is usually per format and not shared even if having same members, think it makes sense to keep things decoupled if things change.

Reply:
It's done